### ISSUE TITLE
TypeError: fitKmerCurve() missing 1 required positional argument: 'adjustment'

### ISSUE THREAD

**Issue by flass** [johnlees/PopPUNK#71]
Hi,

I got another error running PopPUNK; looks like a bug rather than a bad input... can you have a look into that please?

```
Sketching 7628 genomes using 8 thread(s)
Writing sketches to file
Calculating distances using 8 thread(s)
PopPUNK (POPulation Partitioning Using Nucleotide Kmers)
        (with backend: sketchlib v1.1.0)
Mode: Creating clusters from assemblies (create_db & fit_model)
Worst random match probability at 13-mers: 0.03
Problem calculating 0.0
Traceback (most recent call last):
  File "/nfs/users/nfs_f/fl4/software/PopPUNK/poppunk-runner.py", line 10, in <module>
    main()
  File "/nfs/users/nfs_f/fl4/software/PopPUNK/PopPUNK/__main__.py", line 285, in main
    refList, queryList, distMat = queryDatabase(rNames = rNames,
  File "/nfs/users/nfs_f/fl4/software/PopPUNK/PopPUNK/sketchlib.py", line 410, in queryDatabase
    fit = fitKmerCurve(raw, klist, jacobian)
TypeError: fitKmerCurve() missing 1 required positional argument: 'adjustment'
```
the output folder contains only one file:
```
ls -ltr 7kVc/
total 671352
-rw-r--r-- 1 fl4 team216 687456488 Apr  9 21:32 7kVc.h5
```
the command call was:
```
software/PopPUNK/poppunk-runner.py --easy-run --r-files poppunk_7kVc/7kVibrioCholerae_genome_fasta_list.tab --output 7kVc --threads 8 --plot-fit 5 --min-k 13 --max-k 29 --full-db --microreact --ignore-length --estimated-length 4000000
```

I'm using PopPUNK commit 00f4679, which is the tip of the origin/adjusted_distance branch, together with `pp-sketchlib v1.1.0` built from source as reported in https://github.com/johnlees/pp-sketchlib/issues/17.

**Comment by nickjcroucher** [comment]
This should be specific to the adjusted_distance branch, which currently only works with '--use-mash' (as a little experimental) - we're working on integrating this into the sketchlib version, sorry for the confusion.

**Comment by flass** [comment]
OK - I think I'll leave it for a moment so you have the time to integrate all that, and I return for testing of a release version (one that integrate the genome length-related options and uses `pp-sketchlib`).

**Comment by johnlees** [comment]
The error above was fixed in cc4ff02068afef90a52e5ffea5125acf2b9e5d42

**Comment by johnlees** [comment]
@flass Sorry about all this, if you install `PopPUNK v2.0.2` and `sketchlib v1.3.0` everything should be aligned. This version of PopPUNK also fixes the other problem you reported (#70). Please feel free to reopen the issue if there are still issues!

**Comment by flass** [comment]
Hi @johnlees, 
I've successfully installed `PopPUNK v2.0.2` with `pp-sketchlib v1.3.0` and I confirm it works fine - thank you for the patience and work put into the fixes!
now i have to find the right range of parameters...
